### ISSUE TITLE
MACOS15.1.1 has come

### ISSUE THREAD

**Issue by pinkyshit** [assafdori/bypass-mdm#22]
It seems relevant to security

**Comment by Jerry2157** [comment]
I did a clean install and it worked without problems on MacOS 15.1.1

**Comment by pinkyshit** [comment]
But i read that there are some annoying bugs like battery drain faster, some applications failed running etc

**Comment by Jerry2157** [comment]
> But i read that there are some annoying bugs like battery drain faster, some applications failed running etc

Where did you read that?
Let me try it for a week and I'll post my experience here, but for now everything is working fine

**Comment by pinkyshit** [comment]
Well, maybe Reddit, i don't have a clear remember. Yeah, wait for your sharing.

**Comment by dragos420** [comment]
@Jerry2157 any feedback until now? what version (m1-m4)? or it doesn’t matter what silicon it is , just the os version? thanks

**Comment by jwagnervaz** [comment]
> I did a clean install and it worked without problems on MacOS 15.1.1

did you updated to 15.2? 
any problems ?
im at 15.1 without any problem, just waiting to someone confirm sucess to the bypass...

**Comment by Jerry2157** [comment]
> > I did a clean install and it worked without problems on MacOS 15.1.1
> 
> did you updated to 15.2? any problems ? im at 15.1 (M1 Pro) without any problem, just waiting to someone confirm sucess to the bypass...

I'm still at 15.1.1 (M1 Pro) without issues, no battery drain, everything is working fine. I'll not be updating to 15.2, Apple Intelligence is not available in my country so I don't need the 15.2 update, only if a security update is available I will be updating

**Comment by jwagnervaz** [comment]
> > > I did a clean install and it worked without problems on MacOS 15.1.1
> > 
> > did you updated to 15.2? any problems ? im at 15.1 (M1 Pro) without any problem, just waiting to someone confirm sucess to the bypass...
> 
> I'm still at 15.1.1 (M1 Pro) without issues, no battery drain, everything is working fine. I'll not be updating to 15.2, Apple Intelligence is not available in my country so I don't need the 15.2 update, only if a security update is available I will be updating

I will update to 15.2 and leave here my feedback. So much time and people saying no sense things about 15.1 15.1.1 and 15.2

I'm at 15.1 and you 15.1.1 without any problems, so I don't belive these ones talking about not working .

**Comment by jwagnervaz** [comment]
Did update to 15.2 no problems till now, 
As just an update not a clean install.
 Will come back with time and confirm it's OK to update.

**Comment by poporobright** [comment]
> Did update to 15.2 no problems till now, As just an update not a clean install. Will come back with time and confirm it's OK to update.

I did the update as well from 15.0 to 15.2 today, and so far no MDM prompts or locks.

**Comment by jwagnervaz** [comment]
> > Did update to 15.2 no problems till now, As just an update not a clean install. Will come back with time and confirm it's OK to update.
> 
> I did the update as well from 15.0 to 15.2 today, and so far no MDM prompts or locks.

So  mdm bypass is confirmed to work in 15.2

(Did myself too)

Updating from a previous version. 

All this flags is about people that don't know how the run the script...

**Comment by lestitaniseur** [comment]
I have macOS 15.3 can I safely format the Mac then apply the patch 
thanks in advance

**Comment by jwagnervaz** [comment]
> I have macOS 15.3 can I safely format the Mac then apply the patch 
> thanks in advance

No one knows , I recomend you to wait till someone confirm it's working.

**Comment by poporobright** [comment]
> I have macOS 15.3 can I safely format the Mac then apply the patch thanks in advance

Usually the Repo owner would let us know, but some of us have been brave enough to update without wiping the computer first and so far my computer is on 15.2. I updated from 15.0 to 15.2 and no issues whatsoever, up to you to take the risk with these things. Good luck.

**Comment by jwagnervaz** [comment]
> > I have macOS 15.3 can I safely format the Mac then apply the patch thanks in advance
> 
> Usually the Repo owner would let us know, but some of us have been brave enough to update without wiping the computer first and so far my computer is on 15.2. I updated from 15.0 to 15.2 and no issues whatsoever, up to you to take the risk with these things. Good luck.

The repo owner have a life, job, family, open-source sux, people get "our" jobs for free and act like we have any duties with then. 


Crazy times, crazy world.

**Comment by lestitaniseur** [comment]
> > > I have macOS 15.3 can I safely format the Mac then apply the patch thanks in advance
> > 
> > 
> > Usually the Repo owner would let us know, but some of us have been brave enough to update without wiping the computer first and so far my computer is on 15.2. I updated from 15.0 to 15.2 and no issues whatsoever, up to you to take the risk with these things. Good luck.
> 
> The repo owner have a life, job, family, open-source sux, people get "our" jobs for free and act like we have any duties with then.
> 
> Crazy times, crazy world.

I apology 
I just have a question if I reinstalled macOS without data loss the mdm will be reactivated ??

**Comment by poporobright** [comment]
that's a question with risks involved, that until tried we won't know. Why would you want to reinstall Mac OS vs updating it from system settings?

**Comment by poporobright** [comment]
> > > I have macOS 15.3 can I safely format the Mac then apply the patch thanks in advance
> > 
> > 
> > Usually the Repo owner would let us know, but some of us have been brave enough to update without wiping the computer first and so far my computer is on 15.2. I updated from 15.0 to 15.2 and no issues whatsoever, up to you to take the risk with these things. Good luck.
> 
> The repo owner have a life, job, family, open-source sux, people get "our" jobs for free and act like we have any duties with then.
> 
> Crazy times, crazy world.

Right, I don't expect the repo owner to update us daily, he's got stuff to do just like the rest of us, so I try to answer as much as I know and am able to.

**Comment by assafdori** [comment]
You can update to latest firmware freely guys. Only when you factory reset is when you'll be prompted to reconfigure the MDM.

**Comment by jwagnervaz** [comment]
15.3 is beta. So I won't update to beta ones hahaha, I'm so outdated that didn't know 15.3 is beta .
Syndrome of the update
Changelogs 
-Nothing

**Comment by lestitaniseur** [comment]
> You can update to latest firmware freely guys. Only when you factory reset is when you'll be prompted to reconfigure the MDM.

so I can factor reste my Mac to the latest sequoia and the script is working to bypass the mdm at this stage ?
